projects/aca-shared/src/lib/components/toolbar/toolbar-button/toolbar-button.component.ts

```diff
@@ -45,6 +45,20 @@ export enum ToolbarButtonType {
   selector: 'app-toolbar-button',
   templateUrl: './toolbar-button.component.html',
   encapsulation: ViewEncapsulation.None,
+  styles: [
+    `
+      .app-toolbar-button button:focus {
```

Comment:
hardcoded and repetitive values, you should have 1 reusable css class instead of copy-pasting it everywhere

Reply:
Removed this changes